avoidance/launch/avoidance_sitl_mavros.launch

```diff
@@ -24,7 +24,7 @@
         <include file="$(find mavros)/launch/node.launch">
             <arg name="pluginlists_yaml" value="$(find mavros)/launch/px4_pluginlists.yaml" />
             <!-- Need to change the config file to get the tf topic and get local position in terms of local origin -->
-            <arg name="config_yaml" value="$(find local_planner)/resource/px4_config.yaml" />
+            <arg name="config_yaml" value="$(find avoidance)/resource/px4_config.yaml" />
```

Comment:
can you use [this one](https://github.com/mavlink/mavros/blob/master/mavros/launch/px4_config.yaml) instead?

Reply:
Thanks for the feedback. Much better!